Pipfile

```diff
@@ -25,8 +20,11 @@ twine = "*"
 wheel = "*"
 setuptools = "*"
 pypandoc = "*"
-github-stargazers = "*"
-pytest = "*"
```

Comment:
The changes in `Pipfile` are because these should remain the same as it was on master, so these are just to undo the changes from `github-stargazer` branch.